chemreg/substance/tests/factories.py

```diff
@@ -31,13 +35,17 @@ class Meta:
         model = apps.get_model("substance", "SubstanceType")
 
 
-class QCLevelsTypeFactory(ControlledVocabularyFactory):
-    """Manufactures `QCLevelsType` models."""
+class QCLevelsTypeFactory(DjangoSerializerFactory, ControlledVocabularyFactory):
```

Comment:
I like your modification to the ControlledVocabularyFactory here. As long as serializers are still being spat out, I think it would be a good practice to have all of these inherit it so that we're not redefining attributes and repeating code. Would you be ok with updating the other factories as well?

Reply:
Looks like the right thing to do to me.